tests/test_detectionlists_actions.py

```diff
@@ -1,8 +1,6 @@
 from pytest import fixture
 
-from phantom.action_result import ActionResult
-from tests.conftest import create_fake_connector
-from tests.conftest import create_mock_response
+from .conftest import assert_success, create_fake_connector, create_mock_response, assert_successful_single_data, assert_successful_message, assert_succesful_summary
```

Comment:
Why switch to relative import here?
Also could we at least use the tuple syntax here so this isn't going off the screen?

Reply:
I don't think it has to, I think I did this while I was trying to get all the packages properly separated and never came back here.